src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2705,7 +3006,40 @@ private bool DeletePreviousCharacter()
             }
 
             /// <summary>
-            /// Maps point to the current language buffer or standard input buffer.
+            /// Deletes characters succeeding the current caret position in the current language buffer.
+            /// 
+            /// Returns true if the next character was deleted
+            /// </summary>
+            private bool DeleteNextCharacter()
+            {
+                SnapshotPoint? point = MapToEditableBuffer(TextView.Caret.Position.BufferPosition);
+
+                // We are not in an editable buffer, or we are at the end of the buffer, nothing to delete.
+                if (point == null || point.Value == point.Value.Snapshot.Length)
```

Comment:
Consider using `point.GetValueOrDefault()` (after checking `HasValue`) and extracting it into a local.